net/test/node_bulk.c

```diff
@@ -1239,6 +1247,34 @@ static bool node_bulk_bufs_unused_all(struct node_bulk_ctx *ctx)
 	       ctx->nbc_bs_nr;
 }
 
+static void net_bulk_worker_cb(struct node_bulk_ctx *ctx, bool pending)
+{
+	if (USE_LIBFAB) {
+		/* execute network buffer callbacks in this thread context */
+		m0_net_buffer_event_deliver_all(ctx->nbc_net.ntc_tm);
+		M0_ASSERT(ergo(pending, ctx->nbc_callback_executed));
+		/* update copy of statistics */
+		m0_net_test_nh_sd_copy_locked(&ctx->nbc_nh);
+		/* 
+		 * In case of libfabric, there are no pending callbacks,
+		 * hence do not call m0_net_buffer_event_deliver_all() and
```

Comment:
The comment is confusing, you *do* call m0_net_buffer_event_deliver_all().

Reply:
Updated comment